packages/client/components/Avatar/Avatar.tsx

```diff
@@ -1,13 +1,24 @@
-import React, {forwardRef} from 'react'
-import AvatarBadge from '../AvatarBadge/AvatarBadge'
 import styled from '@emotion/styled'
+import React, {forwardRef} from 'react'
 import defaultUserAvatar from '../../styles/theme/images/avatar-user.svg'
+import AvatarBadge from '../AvatarBadge/AvatarBadge'
 
-type ImageBlockProps = Pick<Props, 'sansRadius' | 'sansShadow' | 'picture' | 'size' | 'onClick'>
+type ImageBlockProps = Pick<
+  Props,
+  'sansRadius' | 'sansShadow' | 'picture' | 'size' | 'onClick' | 'isConnected'
+>
 
 const ImageBlock = styled('div')<ImageBlockProps>(
-  ({sansRadius, sansShadow, picture, size, onClick}) => ({
-    backgroundImage: `url(${picture}), url(${defaultUserAvatar})`,
+  ({isConnected, sansRadius, sansShadow, picture, size, onClick}) => ({
+    backgroundImage: `${
+      isConnected || isConnected === undefined
```

Comment:
Copy-paste logic from DashboardAvatar related to 'opacity' for not connected users + backward compatibility for handling components not provided `isConnected` prop  (`undefined`).